broadcast/skins/broadcast/src/coffee/roca.coffee

```diff
@@ -95,7 +113,17 @@
       target.data 'roca-url', url
       target.loading().reload()
       if (el.data 'roca-trap-submit') is 'yes'
-        target.funnelSubmit()
+        options = { "onCompleteEvent": el.data 'roca-submit-complete-event' }
+        target.funnelSubmit(options)
```

Comment:
So, instead of hard-wiring refresh to `funnelSubmit`, make it a generic event handler. Skip passing complete event hame here, and see comment further below.